iDiff/util/tar_utils.go

```diff
@@ -0,0 +1,97 @@
+package tarUtil
+
+import (
+	"archive/tar"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// Dir stores a representaiton of a file directory.
+type Dir struct {
```

Comment:
this same struct is also defined in PR #206 [here](https://github.com/GoogleCloudPlatform/runtimes-common/pull/206/files#diff-4497bc06fb89aa4542205b097595903f), so you should probably get rid of one to avoid code duplication.